Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM jgoerzen/debian-base-standard:stretch
+MAINTAINER bettercities@mysociety.org
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG REPOSITORY_URL_OVERRIDE=git://github.com/mysociety/fixmystreet.git
```

Comment:
This line, and line below, won't be needed in the end.

Reply:
Should we change the MAINTAINER line, too?